docs/pages/versions/unversioned/sdk/calendar.md

```diff
@@ -522,14 +522,26 @@ A person or entity that is associated with an event by being invited or fulfilli
 
 ### RecurrenceRule
 
-A recurrence rule for events or reminders, allowing the same calendar item to recur multiple times.
```

Comment:
Please add documentation to enum types. It can look like [this](https://docs.expo.io/versions/v35.0.0/sdk/location/#enums) (you can skip the `description` column).